qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties

```diff
@@ -1,9 +1,9 @@
-webapp.url=http://localhost:8080/openmrs
+webapp.url=https://demo.openmrs.org/openmrs
```

Comment:
Hello @kdaud  any reason for this change. we had agreed to use local host

Reply:
I was testing carrying out a demo on a certain use case. So the commit has no harm on master!